ios/Classes/SwiftTorusDirectPlugin.swift

```diff
@@ -9,134 +9,239 @@ struct TorusDirectArgs {
+            else {
+                result(FlutterError(
+                        code: "MissingArgumentException",
+                        message: "Missing triggerLogin arguments",
+                        details: nil))
+                return
+            }
+            guard let loginProvider = LoginProviders(rawValue: typeOfLogin) else {
+                result(FlutterError(
+                        code: "InvalidTypeOfLogin",
+                        message: "Invalid type of login",
+                        details: nil))
+                return
+            }
+            
+            let jwtParams = args["jwtParams"] as? Dictionary<String, String>
+            let subVerifierDetails = SubVerifierDetails(
+                loginType: .web,
+                loginProvider: loginProvider,
+                clientId: clientId,
+                verifierName: verifier,
+                redirectURL: initArgs.redirectUri,
+                browserRedirectURL: initArgs.browserRedirectUri,
+                extraQueryParams: [:],
+                jwtParams: jwtParams ?? [:]
+            )
+            let torusDirectSdk = TorusSwiftDirectSDK(
+                aggregateVerifierType: .singleLogin,
+                aggregateVerifierName: verifier,
+                subVerifierDetails: [subVerifierDetails]
+            )
+            torusDirectSdk.triggerLogin(browserType: .external).done { data in
+                result(data)
+            }.catch { err in
+                result(FlutterError())
+            }
+        case "triggerAggregateLogin":
+            guard let initArgs = self.torusDirectArgs
+            else {
+                result(FlutterError(
+                        code: "NotInitializedException",
+                        message: "TorusDirect.init has to be called first",
+                        details: nil))
+                return
+            }
+            guard
+                let aggregateVerifierType = args["aggregateVerifierType"] as? String,
+                let verifierIdentifier = args["verifierIdentifier"] as? String,
+                let subVerifierDetailsArray = args["subVerifierDetailsArray"] as? [Dictionary<String, Any>]
+            else {
+                result(FlutterError(
+                        code: "MissingArgumentException",
+                        message: "Missing triggerAggregateLogin arguments",
+                        details: nil))
+                return
+            }
+            var castedSubVerifierDetailsArray: [SubVerifierDetails] = []
+            for details in subVerifierDetailsArray {
+                guard let loginProvider = LoginProviders(
+                    rawValue: details["typeOfLogin"] as! String
+                ) else {
+                    result(FlutterError(
+                            code: "InvalidTypeOfLogin",
+                            message: "Invalid type of login",
+                            details: nil))
+                    return
+                }
```

Comment:
All error codes are ending with ..Exception, I think this should share the same pattern too.

Reply:
refactored